spicetify-cli.rb

```diff
@@ -1,26 +1,26 @@
-require "language/go"
-
 class SpicetifyCli < Formula
   desc "Command-line tool to customize Spotify client"
   homepage "https://github.com/spicetify/spicetify-cli"
-  url "https://github.com/spicetify/spicetify-cli/archive/v2.24.2.tar.gz"
-  head "https://github.com/spicetify/spicetify-cli"
-  sha256 "7c9b68289e5fe0eef945138a520496f276e03e72395a22b9ab52b5d3ae83ebb8"
+  url "https://github.com/spicetify/spicetify-cli/archive/refs/tags/v2.27.0/v2.27.0.tar.gz"
+  sha256 "824ac8341f5833277a7177a7e17af7036a173e01acbc0730df619c035e0de16b"
+  head "https://github.com/spicetify/spicetify-cli.git", branch: "master"
 
-  depends_on "go" => "1.18"
+  depends_on "go" => :build
```

Comment:
How does this work?

Reply:
The `go.mod` file specifies the version of Go required for the project. In Homebrew formulas, go can be listed as a build dependency, rather than specifying a version, because the version requirement is already clear from go.mod. In theory, it is possible to restate the requirement (see below), but it is redundant.

```
depends_on "go@1.18" => :build
```